RELEASE.md

```diff
@@ -17,6 +17,32 @@ Please follow the established format:
 
 <!-- Add release notes for the upcoming release here -->
 
+# Release 3.8.0
+
+## Major features and improvements
+
+- Finish the new node metadata side panel, which allows you to switch between different modular pipelines. This feature is no longer hidden behind a flag, and the 'meta' flag has been removed. (#293, #309, #312, #320, #321, #326, #295, #329, #333)
+- Enable the new graphing layout algorithm by default, and remove the 'newgraph' feature flag. If necessary, you can still revert back to the old layout algorithm by enabling the 'oldgraph' flag. (#334, #335)
+- Lazy-load sidebar node-list rows on scroll. This will improve performance by not forcing the app to render the entire node-list on larger graphs. This feature is disabled by default, but can be enabled using the 'lazy' feature flag. (#307)
+- Support launching a development server against a real pipeline (#318, #327)
```

Comment:
Would something like this work? 
```suggestion
- Support launching a development server against a real pipeline (#318, #327). This development server is still a work-in-progress, and we will announce when it can be used. 
```

Reply:
Done, I've added something very close to this